mne/channels/tests/test_interpolation.py

```diff
@@ -51,33 +51,21 @@ def test_interplation():
 
     assert_allclose(ave_before, ave_after, atol=2e-6)
 
-    epochs.info['bads'] = []
-    assert_raises(ValueError, epochs.interpolate_bads_eeg)
-
     epochs.info['bads'] = ['EEG 012']
```

Comment:
are you testing MEG interpolation too here? I see only refs to EEG bads in the test